dbt-cta/facebook_marketing/models/1_cta_incremental/_1_cta_incremental__models.yml

```
@@ -425,9 +436,9 @@ models:
         description: ''
         tests:
           - not_null
-          # - relationships:
-          #     to: ref('ad_account_base')
-          #     field: account_id_stripped
+          - relationships:
+              to: ref('ad_account_base')
+              field: account_id_stripped
```

For consistency with the other tests in this file, can we add this config here?

```suggestion
              field: account_id_stripped              
     config:
              where: "created_time > '2022-01-01'"
```

---

added!